doc/update-gh-pages.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Deploy repository documentation
+
+# Variables
+bold=$(tput bold)
+normal=$(tput sgr0)
+script_path="$( cd "$(dirname "$0")" ; pwd -P )"
+project_path=${script_path}/..
+clone_folder=/tmp/update-repos
```

Comment:
Who will create this folder ?

Reply:
And how do you know that this same repository will be in the same path ?

Reply:
> Who will create this folder ?

it is created here: https://github.com/bluerobotics/ping-python/pull/38/files/6ea08ff326be016340e8bd064cf7bb2d7c8338e0#diff-502ee043db64dc7850164e24be6a611cR55

> And how do you know that this same repository will be in the same path ?

 `git clone` will take care of it